docs/source/challenge-doc.rst

```diff
@@ -0,0 +1,415 @@
+MyoChallenge-2024
+#############################################
+
+
+* :ref:`challenge24_manipulation`
+* :ref:`challenge24_locomotion`
+* :ref:`challenge24_tutorial`
+
+
+
+
+.. _challenge24_manipulation:
+
+Prosthesis Co-Manipulation
+--------------------------------------------------------------
+
+Task Overview
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A myoHand :ref:`myoHand` model and a Modular Prosthetic Limb (`MPL <https://www.jhuapl.edu/work/projects-and-missions/revolutionizing-prosthetics/research>`__)
+involved in moving an object between two tables with a handover. This task requires delicate coordination of the 
+object without dropping it or destroying it (maximum force on the object for success) and a mandatory handover between 
+the MyoArm and the MPL to move the objects between two locations.
+
+
+.. image:: images/MyoChallenge-manip.png
+    :width: 450
+    :align: center
+
+
+
+Objective
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Move the object between two locations with a handover between a hand and a prosthesis. The object parameter is randomized in terms 
+of object starting location, object target destination, object weight, and even friction during each environmental reset. 
+
+
+
+Evaluation Criteria
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Task Constrains**
+
+    - The object is within 0.01m from the end goal position
+    - The maximum force exerted upon the object does not exceed 1500N.
+
+
+**Success Criteria**
+
+    - The task is solved
+    - The object first has to touch the MyoArm (100 ms), then the MPL (100 ms), and then the end location (100 ms).
+
+
+The participants will be ranked based on the following criteria hierarchically. 
+
+    1. Finish the task and closeness to the destinations(%)
+    2. Time (s)
+    3. Muscle Effort (-)
+
+.. TODO: can we have how close to the destination here?
+
+
+Action Space
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. TODO: muscle actuator different from locomotion
+
+The action spaces includes two types of actuators. Muscles control values are given as continuous values between  :math:`[0, 1]`, details on how 
```

Comment:
The values and normalisations appears to be different as the locomotion track, needs to check

Reply:
Looks good to me, the prosthetic actuators are normalised to -1 and 1, following the existing normalisation for robotic actuators in the repo.

Reply:
Resolve as it is accurate